spec/elements/table_row_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "spec_helper"
+
+describe OpenXml::Docx::Elements::TableRow do
+  include ElementTestMacros
+
+  context "after creation" do
+    before(:each) do
+      @instance = described_class.new
+      instance.hidden = true
+      cell = OpenXml::Docx::Elements::TableCell.new
+      cell.no_wrap = true
+      instance << cell
+    end
+
+    it_should_output_correct_xml
```

Comment:
We copied this from something that was done with other elements like `Run`. The test compares the output XML to an external file in `/spec/support/data/elements`.